modules/33acrossBidAdapter.js

```diff
@@ -553,14 +617,19 @@ function _isIframe() {
 }
 
 // **************************** INTERPRET RESPONSE ******************************** //
-// NOTE: At this point, the response from 33exchange will only ever contain one bid
-// i.e. the highest bid
 function interpretResponse(serverResponse, bidRequest) {
   const bidResponses = [];
-
-  // If there are bids, look at the first bid of the first seatbid (see NOTE above for assumption about ttx)
-  if (serverResponse.body.seatbid.length > 0 && serverResponse.body.seatbid[0].bid.length > 0) {
-    bidResponses.push(_createBidResponse(serverResponse.body));
+  const { seatbid, cur } = serverResponse.body;
+
+  if (utils.isArray(seatbid)) {
+    seatbid.forEach(function(seat) {
+      if (utils.isArray(seat.bid)) {
+        seat.bid.forEach(function(bid) {
```

Comment:
qq.
is there any reason in particular why the `forEachs` use regular functions instead of arrow functions?

Probably `this` will never be used, but at the same time I don't see the point of using a loop function that by default offers 'constructor' support when there is no need for it.

Reply:
No reason at all, I will fix this.